docs/public-dns/solving-problems/how-to-flush-dns-cache.md

```diff
@@ -50,9 +50,27 @@ By doing that, you will lose connections to Wi-Fi routers and other specific net
 
 ### Android
 
-The easiest way to clear your DNS cache on your Android device is to turn the Airplane mode on and off. You can enable/disable the Airplane Mode in the Quick Settings pane.
+There are different ways to clear the DNS cache on your Android device. It's important to note that the exact steps may vary depending on the version of Android you're using and the device manufacturer.
 
-A hard reboot of the device also flushes the DNS cache. In order to do that, press and hold the power button for at least 20 seconds. It will (usually) force your device to reboot manually and the DNS cache will be cleared.
+#### Clear DNS cache via Chrome
+
+Google Chrome, often the go-to browser on Android, holds its own DNS cache. To flush this cache in the Chrome browser, follow the instructions below:
```

Comment:
Google Chrome, often the default browser on Android, has its own DNS cache. To flush this cache in the Chrome browser, follow the instructions below: